<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1532</width>
    <height>973</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1829</width>
     <height>929</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="ViewerLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1532</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarFile">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile_2"/>
   <addaction name="actionDeleteFile"/>
   <addaction name="actionSaveFile_2"/>
  </widget>
  <widget class="QToolBar" name="toolBarViewer">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFocuse"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionCenterModel"/>
   <addaction name="actionScreenShot"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidgetLog">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QTextBrowser" name="textBrowser">
       <property name="minimumSize">
        <size>
         <width>1600</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="PushButtonClearLog">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PushButtonSaveLog">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetSettings">
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>524287</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeWidget" name="treeWidgetModels">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>300</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>Models</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="StatusLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/download12/folder-open.svg</normaloff>:/icons/download12/folder-open.svg</iconset>
   </property>
   <property name="text">
    <string>OpenFile</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/download12/save.svg</normaloff>:/icons/download12/save.svg</iconset>
   </property>
   <property name="text">
    <string>SaveFile</string>
   </property>
  </action>
  <action name="actionCenterModel">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/download12/checkbox.svg</normaloff>:/icons/download12/checkbox.svg</iconset>
   </property>
   <property name="text">
    <string>CenterModel</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Center model in display&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionOpenFile_2">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/download12/file-addition-one.svg</normaloff>:/icons/download12/file-addition-one.svg</iconset>
   </property>
   <property name="text">
    <string>OpenFile</string>
   </property>
  </action>
  <action name="actionDeleteFile">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/download12/file-removal-one.svg</normaloff>:/icons/download12/file-removal-one.svg</iconset>
   </property>
   <property name="text">
    <string>DeleteFile</string>
   </property>
  </action>
  <action name="actionSaveFile_2">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/download12/save.svg</normaloff>:/icons/download12/save.svg</iconset>
   </property>
   <property name="text">
    <string>SaveFile</string>
   </property>
  </action>
  <action name="actionFocuse">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/download12/aiming.svg</normaloff>:/icons/download12/aiming.svg</iconset>
   </property>
   <property name="text">
    <string>Focuse</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/download12/zoom-in.svg</normaloff>:/icons/download12/zoom-in.svg</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/download12/zoom-out.svg</normaloff>:/icons/download12/zoom-out.svg</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
  </action>
  <action name="actionScreenShot">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/download12/camera.svg</normaloff>:/icons/download12/camera.svg</iconset>
   </property>
   <property name="text">
    <string>ScreenShot</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
